docs/reference/cluster/update-settings.asciidoc

```diff
@@ -46,42 +53,51 @@ value. Here is an example:
 
 [source,js]
 --------------------------------------------------
-curl -XPUT localhost:9200/_cluster/settings -d '{
+PUT /_cluster/settings
+{
     "transient" : {
-        "discovery.zen.minimum_master_nodes" : null
+        "indices.recovery.max_bytes_per_sec" : null
     }
-}'
+}
 --------------------------------------------------
+// CONSOLE
+// TEST
 
 Reset settings will not be included in the cluster response. So
 the response for the last example will be:
 
 [source,js]
 --------------------------------------------------
 {
+    ...
     "persistent" : {},
     "transient" : {}
 }
 --------------------------------------------------
+// CONSOLE
```

Comment:
This will cause the response to have `COPY AS CURL` and `VIEW IN CONSOLE` links which is not right. It is fine to leave it off.